charts/argo-cd/values.yaml

```diff
@@ -595,6 +595,14 @@ server:
   ## Annotations to be added to ArgoCD rbac ConfigMap
   rbacConfigAnnotations: {}
 
+  # Name of the configmap used for the RBAC config. Must be set to `argocd-rbac-cm` as the values are hardcoded in
+  # ArgoCD. See https://github.com/argoproj/argo-cd/blob/master/common/common.go
+  rbacConfigMapName: "argocd-rbac-cm"
+
+  # Boolean determining whether or not to create the configmap. If false, it is expected tthe configmap will be created
+  # by something else. ArgoCD will not work if there is no configMap created with the name above.
+  rbacConfigMapEnabled: true
```

Comment:
I would name this value rbacConfigCreate to be coherent with the controller.serviceAccount.create value (we cannot have rbacConfig.create since rbacConfig is already used). What do you think?